test/harness.ts

```diff
@@ -393,9 +393,9 @@ expect.extend({
       message: () => `Expected ${cmds.join(" ")} to fail`,
     };
   },
-  toThrowWithCode(fn: CallableFunction, cls: CallableFunction, code: string) {
+  async toThrowWithCode(fn: CallableFunction, cls: CallableFunction, code: string) {
```

Comment:
is this necessary? and if it is necessary, is it always necessary? I don't trust we'll remember to `await` these expect calls.